test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -469,6 +469,10 @@ public void TestPfiRankingOnDenseFeatures(bool saveModel)
 
                 var loadedModel = ML.Model.Load(modelAndSchemaPath, out var schema);
                 var castedModel = loadedModel as RankingPredictionTransformer<FastTreeRankingModelParameters>;
+
+                // Saving and Loading the model cause the internal random state to change, so we reset the seed
+                // here so help the tests pass.
```

Comment:
Interesting. I wonder what's different?

Reply:
Not sure. Harish and I think that save/load shouldn't change the random state, so there is a bug there, but not one that needs to be solved with this PR. I am going to open an issue so we can track it.

Reply:
I fully agree; it's outside the scope of this PR.

Reply:
Save and load follow different code paths which changes the state of the random number generator when registering with the environment. This can be fixed if we fix how the random state is managed. But that is a fairly deep change and will affect the baseline of every test we have. 

---
In reply to: [541290600](https://github.com/dotnet/machinelearning/pull/5433#discussion_r541290600) [](ancestors = 541290600)